### DISCUSSION TITLE
Alicevision version greater than 1.2.6

### DISCUSSION THREAD

**Discussion by wziard** [SBCV/Blender-Addon-Photogrammetry-Importer#72] [General]
In meshroom_file_handler.py there is a version check for the meshroom version being smaller than 1.2.6

If I comment out this check it seems to load Meshroom-2023.3.0 project just fine, except for one thing:
 if I enable 'Adjust Frame Numbers of Camera Animation' it will try to  find the unreconstructed cameras by listing all images in the original image directory. But since this listing is recursive (and it needs to be, because the input images can be spread about multiptl subdirs?)  it picks up all Meshroom generated images ( .exr depthmaps, undistored  images, filtered depthmaps, everything) as well.

**Comment by SBCV** [comment]
Thank you for reporting this. In cases like this, its  probably the best idea to directly open an issue.
Created an issue [here](https://github.com/SBCV/Blender-Addon-Photogrammetry-Importer/issues/74) and will investigate it further in the coming weeks.
